data_preparation/data_cleaning.py

```diff
@@ -278,3 +278,10 @@ def fix_int_datatypes(df):
                         'co2_emissions': 'int', 'total_reviews': 'int'})
                         
     return df
+
+
+if __name__ == '__main__':
+    df = pd.read_csv('/Users/brook/downloads/data.csv')
```

Comment:
I think this line is too specific to your file structure, could it be written so i could run it with the data.csv file in the top level folder?

Reply:
`SEE FINAL FILE VERSION`